liquidctl/driver/msi_7zw0led.py

```diff
@@ -0,0 +1,43 @@
+"""liquidctl drivers for MSI liquid coolers.
+
+Supported devices:
+
+- MPG CORELIQUID K360
+
+Copyright (C) 2020–2021  Andrew Udvare and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+from liquidctl.driver.usb import UsbHidDriver
+
+_REPORT_LENGTH = 64
+
+
+class MSI7ZW0LED(UsbHidDriver):
+    """7ZW0LED is the internal name in the MSI Center app."""
+
+    SUPPORTED_DEVICES = [
+        (0x0db0, 0xb130, None, 'MSI MPG Series CORELIQUID K360', {}),
+    ]
+
+    def get_status(self, **kwargs):
+        buf = bytearray(_REPORT_LENGTH)
+        buf[1] = 0xd0
+        buf[2] = 0x31
+        self.device.clear_enqueued_reports()
+        self.device.write(buf)
+        array = bytes(self.device.read(_REPORT_LENGTH))
+        if array[1] != 0x31:
+            raise ValueError('Unexpected value in response buffer')
+        return [
+            ('Fan 1 speed', array[2] + (array[3] << 8), 'rpm'),
+            ('Fan 1 duty', array[0x16] + (array[0x17] << 8), '%'),
+            ('Fan 2 speed', array[4] + (array[5] << 8), 'rpm'),
+            ('Fan 2 duty', array[0x18] + (array[0x19] << 8), '%'),
+            ('Fan 3 speed', array[6] + (array[7] << 8), 'rpm'),
+            ('Fan 3 duty', array[0x1A] + (array[0x1B] << 8), '%'),
+            ('Water block fan speed', array[8] + (array[9] << 8), 'rpm'),
+            ('Water block fan duty', array[0x1C] + (array[0x1D] << 8), '%'),
+            ('Pump fan speed', array[0xA] + (array[0xB] << 8), 'rpm'),
+            ('Pump fan duty', array[0x1E] + (array[0x1F] << 8), '%'),
```

Comment:
Shouldn't these be "Pump speed" and "Pump duty" (no "fan")?

Reply:
IIRC as I cannot see it right now, the MSI UI says 'Pump Fan' and 'Water Block Fan'. Maybe this is a mistake on their part and I'm happy to correct it. Of course there is no physical 'pump fan' or 'water block fan'. As for the speed reported, I don't know what it means.